public/data/AntonyKimani.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "Antony Kimani",
+    "type": "personal",
+    "bio": "Frontend Web Developer | Open Source Enthusiast | Blogger ",
```

I think this space should be removed 😅 What do you think?
```suggestion
    "bio": "Frontend Web Developer | Open Source Enthusiast | Blogger",
```